_posts/2019-10-14-hhvm-4.27.0.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: "HHVM 4.27.0"
+layout: post
+author: fred 
+category: blog
+---
+
+HHVM 4.27 is released! This release marks the end of support for 4.21;
+4.22&ndash;4.26 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- Clearer typechecker error message when a function does not explicitly
+  return a value in all branches
+- Added 'jump-to-definition' for overridden methods
+- go-to-implementation ("textDocument/implementation") is now implemented in
+  LSP-based editors (e.g. Visual Studio Code) for common situations.
+- added `set_pre_timeout_handler(int $seconds, 
```

Comment:
missing end of the header

Reply:
whoops, leaving to next week for better HHI